desktop/release-notes.md

```diff
@@ -24,12 +24,100 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.14
+
+2022-11-10
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
```

Comment:
Can the download links and the SHA values be added to this page yet @ebriney? 

All of the pages under 'Installation per linux distro will also need to have the download links changed.

Reply:
You're alright, I need to modify the SHAs there

Reply:
and linux links as well, my bad

Reply:
done